tensorflow/contrib/nodejs/test/suite.js

```diff
@@ -0,0 +1,7 @@
+var test = require('tape')
+  , tensorflow = require('../index')
```

Comment:
For Python, we are generally following Google style guide, but are using the naming conventions from PEP8, to be more consistent with open-source users. So Google style guide is a good default, but if there are reasons to use a different style guide that's more familiar or widely adopted in the nodejs community I think we'd be open to that - especially since I doubt think the node bindings will see much use inside of Google. 

If you think the google js style guide is a good choice then let's go with that.

@martinwicke feel free to comment more authoritatively... (since I know you have such passionate, passionate thoughts on javascript styles)